src/main/java/frc/robot/RobotContainer.java

```diff
@@ -141,7 +147,14 @@ public RobotContainer() {
     } else {
       turretSubsystem = null;
     }
-    turretCommandFactory = new TurretCommandFactory(turretSubsystem);
+    turretCommandFactory =
+        new TurretCommandFactory(
+            turretSubsystem,
+            robotStateManager,
+            visionSubsystem,
+            drivetrain::getRotation,
+            drivetrainCommandFactory::currentRobotPosition);
+    SmartDashboard.putData(turretCommandFactory.zeroZeroing());
```

Comment:
Earlier in the season commands from SHuffleboard were not working.  Are they working now?

Do we expect to use this command in competition?  Let's put it on a specific tab for competition, or block it behind an if ( isCompetition)